src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -2845,9 +2905,11 @@ def multiply_by_conjugate(self, J):
         R = self.quaternion_algebra()
         return R.ideal(basis, check=False)
 
-    def is_equivalent(self, J, B=10) -> bool:
-        """
-        Return ``True`` if ``self`` and ``J`` are equivalent as right ideals.
+    def is_equivalent(self, J, B=10, certificate=False, side=None):
```

Comment:
Personally I would simply deprecate this method in favor of the two new explicitly named one-sided methods: Having it default to `.is_right_equivalent()` is arbitrary and ambiguous.

Reply:
So should I add a deprecation warning for now to set it up for future removal, or should I remove it immediately?

Reply:
According to The Rules, we're only allowed to remove things once they've been deprecated for one year, so it has to be the former: Add a deprecation warning. (If you agree, that is: I don't have a strong opinion about this either way.)